serving/docker/Dockerfile

```diff
@@ -13,8 +13,7 @@ FROM ubuntu:20.04 AS base
 ARG djl_version=0.23.0~SNAPSHOT
 
 COPY scripts scripts/
-RUN mkdir -p /opt/djl/conf && \
```

Comment:
I think empty model_store directory is a validate use case. We can keep the folder here. I will create a separate PR to fix empty model_store bug.